### PR TITLE
Add double hashed indexer - inga

### PR DESCRIPTION
**Pull request by ischasny**
`inga` is a new production double hashed indexer that writes into production `dhstore`. In this PR:
* create a new node group with the same instance type that is used by `ago` in dev;
* create a new identity for `inga`;
* create a new deployment configuration for `inga`. Note - `inga` won't be mirroring car files into s3 a this is being already done by `ago`.

### CODE REVIEW

File: deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/inga/pvc_addata.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+kind: PersistentVolumeClaim
+metadata:
+  name: addata
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 3Ti
+  storageClassName: io2
```
Comment (masih) [inline deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/inga/pvc_addata.yaml:11]: Let's use `gp3` volumens for both data and ad data

File: deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/inga/config.json [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+      "basichost": "warn",
+      "bootstrap": "warn",
+      "dt-impl": "warn",
+      "dt_graphsync": "warn",
+      "graphsync": "warn"
+    }
+  },
+  "Peering": {
+    "Peers": [
+      "/dns4/assigner/tcp/3003/p2p/12D3KooWDBjcDRQ7CKJeF9Yy3UKbriHfyETDrXKzTDB6biH3ibBd"
```
Comment (masih) [inline deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/inga/config.json:103]: THis node is not being assigned to. Right?

Can you check if assigner is configured to propagate all announcements over gossipsub? If so then peering makes sense.

Comment (ischasny) [reply deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/inga/config.json:103]: Ha, interesting. Ago has got the same config.

File: deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/inga/pvc_addata.yaml (hunk 1 above)
Comment (ischasny) [reply deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/inga/pvc_addata.yaml]: Changed

File: deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/inga/config.json (hunk 2 above)
Comment (ischasny) [reply deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/inga/config.json]: Removed

Comment (masih) [reply deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/inga/config.json]: It is harmless to leave this be. my comment was more abut understanding its usefulness.

File: deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/README.md [hunk 3]
```diff
@@ -7,3 +7,4 @@ List of individually configurable instances:
 | `oden`   | io2           | 5            | Pebble        | [v0.5.4](https://github.com/filecoin-project/storetheindex/releases/tag/v0.5.4) |
 | `dido`   | io2           | 5            | Pebble        | [v0.5.4](https://github.com/filecoin-project/storetheindex/releases/tag/v0.5.4) |
 | `kepa`   | io2           | 3            | Pebble        | [v0.5.4](https://github.com/filecoin-project/storetheindex/releases/tag/v0.5.4) |
+| `inga`   | io2           | -            | DHStore       | [v0.5.4](https://github.com/filecoin-project/storetheindex/releases/tag/v0.5.4) |
```
Comment (masih) [inline deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/README.md:10]:
```suggestion
| `inga`   | gp3           | -            | DHStore       | [v0.5.4](https://github.com/filecoin-project/storetheindex/releases/tag/v0.5.4) |
```

File: deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/inga/config.json (hunk 2 above)
Comment (ischasny) [reply deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/inga/config.json]: Ok I'll remove the commit

File: deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/inga/pvc_data.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    app: indexer-single
+    app.kubernetes.io/managed-by: kustomization
+    app.kubernetes.io/part-of: storetheindex
+  name: data
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Ti
```
Comment (masih) [inline deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/inga/pvc_data.yaml:14]: ago disk on dev is `0.1`% utilised. I think we can reduce this more and expand as needed. Let's start with with 10Gi

**Review by masih — APPROVED** [review]

File: deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/inga/pvc_data.yaml (hunk 4 above)
Comment (ischasny) [reply deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/inga/pvc_data.yaml:14]: Done

**Pull request merged**